code/DefaultMembers.php

```diff
@@ -73,4 +81,39 @@ public static function create_default_members($silent=FALSE) {
         }
     }
 
+    /**
+     * Remove default members
+     *
+     * @param bool $silent Pass true to hide `/dev/build` messages
+     */
+    public static function remove_default_members($silent = false)
+    {
+        $remove = Config::inst()->get('DefaultMembers', 'remove');
+
+        if ($remove && is_array($remove)) {
+            foreach ($remove as $data) {
+
+                if (!Email::is_valid_address($data['Email'])) {
+                    continue;
+                }
```

Comment:
This isn't necessary because we're not writing to the DB.  It's possible to have Members with bad/invalid emails that someone may want to use this extension to help purge.